source/postprocess/gravity_point_values.cc

```diff
@@ -421,12 +429,13 @@ namespace aspect
       {
         prm.enter_subsection ("Gravity calculation");
         {
-          prm.declare_entry ("Sampling scheme", "map",
-                             Patterns::Selection ("map|list"),
-                             "Choose the sampling scheme. A map will produce a grid of "
-                             "equally spaced points between "
-                             "a minimum and maximum radius, longitude, and latitude. A "
-                             "list contains specific coordinates of the satellites.");
+          prm.declare_entry ("Sampling scheme", "uniform distribution",
+                             Patterns::Selection ("uniform distribution|list of points"),
```

Comment:
This is not backwards compatible.  Was this postprocessor already part of the last release? If yes, I would vote to also leave the old names in here and just set them to the corresponding new enum value.

Reply:
May I ask why it is not backwards compatible? So far I have been the only one using this postprocessor and there are no benchmarks or cookbooks using it. 

An alternative could then be to have a "profile" sampling scheme as I initially planned and keeping map and list, but which is nothing else than the "map" sampling scheme. For me the main issue is the name "map which is too restrictive while the postprocessor can actually do more.

Reply:
I don't feel too strongly about it, but I don't think we can be sure nobody has used it so far. Maybe lets see if anybody else has an opinion. It is not hard to add the 2 lines of code, I would think.